transit/rest_api/excel/example_excel_data.py

```diff
@@ -0,0 +1,138 @@
+CUSTOMER_EXAMPLE_DATA = [{
```

Comment:
@dborowiecki I decided to not take data from 'test' to not mix 'test' logic with standard endpoint implementation. 
15 code smells are related to this example excel data helper. Let me know whether should I fix this or keep like this.